### PR TITLE
A script to install a ZIS plugin from file

### PR DESCRIPTION
**Pull request by fkovinAtRocket**

### CODE REVIEW

**Comment by fkovinAtRocket** [comment]
@ifakhrutdinov @1000TurquoisePogs Could you please take a look?

File: bin/zis-plugin-install.sh [hunk 1]
```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
+
+ZIS_PREFIX=${ZIS_PREFIX:-$USER.DEV}
+ZIS_PARMLIB=${ZIS_PARMLIB:-$ZIS_PREFIX.PARMLIB}
+ZIS_LOADLIB=${ZIS_LOADLIB:-$ZIS_PREFIX.LOADLIB}
+
+mktemp1()
+{
+  FILE=/tmp/nwt-$$-$RANDOM
```
Comment (ifakhrutdinov) [inline bin/zis-plugin-install.sh]: What does ```nwt``` stand for? Can we change this to something zss specific?

File: bin/zis-plugin-install.sh [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  chmod go-rwx $TMPFILE
+
+  if add-plugin-to-parmfile $TMPFILE
+  then
+    >&2  echo "Updating $ZIS_PARMLIB(ZWESIP00)..."
+    cp -v $TMPFILE "//'$ZIS_PARMLIB(ZWESIP00)'"
+  fi
+  
+  >&2 echo "Installing plugin $LMODFILE to $ZIS_LOADLIB($LMODNAME) and $ZIS_PARMLIB"
+  cp -v $LMODFILE "//'$ZIS_LOADLIB'"
```
Comment (ifakhrutdinov) [inline bin/zis-plugin-install.sh]: Can we change this to 
```
cp -X -v $LMODFILE "//'$ZIS_LOADLIB'" 
```
See https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.1.0/com.ibm.zos.v2r1.bpxa400/cpexfile.htm. We fixed a similar bug in the zowe install recently.

**Comment by ifakhrutdinov** [comment]
@1000TurquoisePogs , @fkovinAtRocket is on vacation, is it okay if someone else will make the requested changes?

**Comment by ifakhrutdinov** [comment]
Also, is there a specific reason to have this file in ```bin``` and not in ```build```?

**Comment by fkovinAtRocket** [comment]
> Also, is there a specific reason to have this file in `bin` and not in `build`?

Yes, this is not used when building ZSS. Rather, the user will call this to install a ZIS plugin into an already existing installation.

**Comment by ifakhrutdinov** [comment]
Basic tests pass. LGTM.

**Pull request merged**